{% extends "layout.html" %}

{% block page_author %}Weiwen CHEN{% endblock %}
  
{% block title %}Engineering Success{% endblock %}
{% block lead %}Demonstrate engineering success in a part of your project by going through at least one iteration of the engineering design cycle.{% endblock %}

{% block side_bar %}
<div id="landingpage" style="width:20%">
          <ul>
          <li>
            <div class="layer">
              <div class="anchor">
                <a class="dot-landingpage" data-scroll="content" href="#ribozyme-assisted-polycistronic-co-expression-system-prap">
                <span class="linkname">Polycistronic expression</span></a>
                </div>
              <img src="https://static.igem.wiki/teams/4162/wiki/arrow-orange.png" class="arrow" style="cursor:pointer" />
            </div>
        <ul class="submenu">
            <li>
                <a class="anchor linkname" href="#ribozyme-assisted-polycistronic-co-expression-system-prap">Polycistronic expression</a>
            </li>
            <li>
                <a class="anchor" href="#cycle-1">Cycle 1</a>
            </li>
            <li>
                <a class="anchor" href="#cycle-2">Cycle 2</a>
            </li>
        </ul>
          </li>
          <li>
            <div class="layer">
              <div class="anchor">
                <a class="dot-landingpage" data-scroll="content1" href="#application-of-hplc-in-product-detection">
                <span class="linkname">HPLC for product validation</span></a>
                </div>
              <img src="https://static.igem.wiki/teams/4162/wiki/arrow-orange.png" class="arrow" style="cursor:pointer" />
            </div>
        <ul class="submenu">
            <li>
                <a class="anchor linkname" href="#application-of-hplc-in-product-detection">HPLC for product validation</a>
            </li>
            <li>
                <a class="anchor" href="#cycle-1-1">Cycle 1</a>
            </li>
            <li>
                <a class="anchor" href="#cycle-2-1">Cycle 2</a>
            </li>
        </ul>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content2" href="#references">
                <span class="linkname">References</span></a>
              </div>
          </li>
          </ul>
</div>
{% endblock %}
{% block page_content %}

<!-- ADD highlight pictures BELOW -->
<div class="row">
<br/><br/>
</div>

<!-- ADJUST PAGE CONTENT BELOW -->

<div class="row">
  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content">
<p>In the process of building single-cell factory, we encountered several problems in <a href="/fudan/parts">parts</a> design and experiments, and tried to solve them following iterations of the <strong>DBTL</strong> cycle (Design, Build, Test, Learn).</p>

<h2 id="ribozyme-assisted-polycistronic-co-expression-system-prap">Ribozyme-assisted polycistronic co-expression</h2>
<h3 id="cycle-1">Cycle 1</h3>

<h4 id="design">Design</h4>
<p>As many researches indicate, the major problem of polycistronic vectors, which contain two or more target genes under one promoter, is the much lower expression of the downstream genes compared with that of the first gene next to the promoter (Kim, Kyung-Jin et al. 2004). The tail of the coding sequence (CDS) can interfere with the head of the ribosome binding site (RBS), which can hinder RBS from combining to ribosomes. Such shortage occurred when we assembled crtEBIY sequentially only to find incomplete expressions of our target proteins.</p>
<p>To optimize the crt BioBrick, we adopted hammerhead ribozyme <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162005">BBa_K4162005</a> to construct a ribozyme-assisted polycistronic co-expression BioBricks. Through inserting ribozyme sequences between crtEBIY, the polycistronic mRNA transcript is thus co-transcriptionally converted into individual mono-cistrons which have comparable translation efficiency. By building this system, we expected that the self-interaction of the polycistron can be avoid.</p>

<h4 id="build">Build</h4>
<p>We first assemble four basic modules: <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162010">BBa K4162010</a> (ribozyme + T7_RBS + crtE), <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162013">BBa_K4162013</a> (ribozyme + T7_RBS + crtB), <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162016">BBa_K4162016</a> (ribozyme + T7_RBS + crtI) and <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162019">BBa_K4162019</a> (ribozyme + T7_RBS + crtY). Next, we apply overlapping PCR to assemble 2-3 random basic modules. Finally, we obtained several BioBricks composed of four basic modules assembled in different orders.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/crt-biobrick.png" alt="crt-biobrick.png (5827×1260) (igem.wiki)"></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 1.</strong> Module crt IYEB <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162021">BBa_K4162021</a>, an example of composite modules which contain four basic modules.</p>

<h4 id="test">Test</h4>
<p>We tested the protein expressions of different combinations of random carotene synthesis enzymes through SDS-PAGE, and as Figure 2 and 3 indicate, each enzyme can be expressed successfully.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/3e.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 2.</strong> IPTG(-/+) = without/with 0.2 mM IPTG for 3-6 hours, adding IPTG to a bacteria culture with OD600 0.2-0.3. M: Protein molecular weight marker ladder. Lane 1~4, 6~9, 11~17: pET28 plasmids encoding crtYB, IB, EY separated by self-cleaving ribozyme, crtB, crtE, crtY without any tag were transformed into BL21(DE3) HI-Control strain, single clones (1YB1/IB7/IB8/1B1/EY3/EY4/1E1/1Y1) were picked for liquid LB culture. Lane 10: pET28 plasmids encoding crtI were transformed into BL21(DE3) HI-Control strain, single clones (1I7) were picked for liquid LB culture. Protein expression was induced in parallel cultures by IPTG. Bacterial cultures were monitored by OD600, and 5x10^7 cells were harvested by centrifugation and lysis in 1x SDS sample buffer. Equal amount (10 μL, 2x10^6 cells) of whole cell lysate were analyzed by SDS-PAGE (4~20% gradient gel, Tanon brand). Red arrows point to crtI protein. Green arrows point to crtY protein. Black arrows point to crtB protein. Yellow arrows point to crtE protein.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/3f.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 3.</strong> IPTG(-/+) = without/with 0.2 mM IPTG for 3-6 hours, adding IPTG to a bacteria culture with OD600 0.2-0.3. M: Protein molecular weight marker ladder. Lane 4~16: pET28 plasmids encoding crtBEI separated by self-cleaving ribozyme, crtI, crtEI, crtB, crtEB, crtIYEB separated by self-cleaving ribozyme without any tag were transformed into BL21(DE3) HI-Control strain, single clones (BEI/1I7/1EI/1B1/EB5/EB6/IYEBa) were picked for liquid LB culture. Lane 1~2: pET28 plasmids encoding BCMO-ybbO-tdMCP-EGFP were transformed into BL21(DE3) HI-Control strain, single clones (2BY14tG14) were picked for liquid LB culture. Lane 3~4: pET28 plasmids encoding BCMO-ybbO were transformed into BL21(DE3) HI-Control strain, single clones (BY14hi) were picked for liquid LB culture. Protein expression was induced in parallel cultures by IPTG. Bacterial cultures were monitored by OD600, and 5x10^7 cells were harvested by centrifugation and lysis in 1x SDS sample buffer. Equal amount (10 μL, 2x10^6 cells) of whole cell lysate were analyzed by SDS-PAGE (4~20% gradient gel, Tanon brand). Red arrows point to crtI protein. Green arrows point to crtY protein. Black arrows point to crtB protein. Yellow arrows point to crtE protein.</p>

<h4 id="learn">Learn</h4>
<p>We constructed successfully enabled the expression of each enzyme contained in the crt polycistron in comparable proportions. However, if one wants to precisely manage the proportion of expression products of individual CDSs in this polycistron, it is not enough to separate the CDSs by ribozyme sequences. Besides, the stablility of expression among clones cannot be confirmed by electrophoresis experiments.</p>

<h3 id="cycle-2">Cycle 2</h3>
<h4 id="design-1">Design</h4>
<p>Next, we designed composite modules adjusted for the RBS of individual enzymes and performed incubation and centrifugation experiments to probe the stability of expression.</p>

<h4 id="build-1">Build</h4>
<p>Typical modules we designed are <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162117">BBa_K4162117</a> and <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162118">BBa_K4162118</a>.</p>
<p>BBa_K4162117: This BioBrick was created through overlapping PCR of <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162020">BBa_K4162020</a> (ribozyme+J6_RBS+crtY), <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162010">BBa_K4162010</a> (ribozyme+T7_RBS+crtE), <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162013">BBa_K4162013</a> (ribozyme+T7_RBS+crtB) and <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162016">BBa_K4162016</a> (ribozyme+T7_RBS+crtI). In this part, the RBS of crtEBI has equal intensity while the RBS of crtY is significantly weaker than the others. Because crtY catalyzes the last step of the carotenoid reaction chain, we guess the concentration of substrate catalyzed by this enzyme is significantly lower than for the first three steps of the reaction. To avoid the problem of flux imbalance in biosynthesis as well as to reduce unnecessary metabolic stress on cells, we intentionally weakened the RBS intensity of crtY.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/yebi.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 4.</strong> Module crtYEBI, using a <a target="_blank" href="http://parts.igem.org/Part:BBa_J61100" target="_blank">weak</a> RBS drives crtY expression.</p>

<p>BBa_K4162118: This BioBrick was created through overlapping PCR of <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162020">BBa_K4162020</a> (ribozyme+J6_RBS+crtY), <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162010">BBa_K4162010</a> (ribozyme+T7_RBS+crtE), <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162013">BBa_K4162013</a> (ribozyme+T7_RBS+crtB) and <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162016">BBa_K4162016</a> (ribozyme+T7_RBS+crtI). In this part, the RBS of crtBIY has equal intensity while the RBS of crtE is significantly stronger than the others. Since crtE catalyzes the first step of the carotenoid reaction chain, increase the concentration of product catalyzed by this enzyme is beneficial for the remaining three steps. To avoid more serious flux imbalance problems, we boosted the RBS intensity of crtE only in this BioBrick and explored whether the carotenoid production of the strain could be significantly enhanced.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/ebiy.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 5.</strong> Module crtEBIY, using a <a href="http://parts.igem.org/Part:BBa_B0030" target="_blank">strong</a> RBS drives crtE expression.</p>

<h4 id="test-1">Test</h4>
<p>We transformed plasmids into <em>E. coli</em> DH5&alpha; and cultured it, and finally obtained YELLOW bacterial pellet after centrifugation.</p>
<p>Figures 6 to 8 show that transformed <em>E. coli</em> successfully expressed the target enzyme and yielded &beta;-carotene. In Figure 6, it can be seen that module YEBI corresponds to a darker orange color of the post-centrifugation precipitation compared to module YBEI (<a target="_blank" href="http://parts.igem.org/Part:BBa_K4162119">BBa_K4162119</a>), characterizing the superior carotenoid yielding ability of module YEBI. As shown in Figure 8, different clones of <em>E. coli</em> transformed with the same plasmid were cultured using 96-well plates. After shaking culture overnight, the plate was spin at 4500 rpm for 30 minutes and bacteria was pelleted to the bottom of the wells. There is no clonal variation, showing stable expression of the individual mono-cistrons separated by ribozyme sequences (i.e., no trucated expression nor null function protein of downstream genes due to being placed under one promoter).</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/117-119.jpg" alt="117-119.jpg (3840×2160) (igem.wiki)"></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 6.</strong> The centrifuge tubes containing module <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162119">crtYEBI</a> (first from the left) and module crtYBEI <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162119">BBa_K4162119</a> (second from the left) contain visible yellow bacterial pellet.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/118.jpg" alt="118.jpg (2160×3840) (igem.wiki)"></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 7.</strong> Overnight culture of bacterial expressing module <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162118">crtEBIY</a> was centrifuged at 13000 rpm for 1 minute.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/96a-117.jpg" alt="96a-117.jpg (3840×2160) (igem.wiki)"></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 8.</strong> <strong>96-well plate of module <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162117">crtYEBI</a>.</strong> Except for the blank control well marked in black, all clones growing different wells had similar &beta;-carotene content in the bacterial pellet.</p>
<h4 id="learn-1">Learn</h4>
<p>Multiple cis-trans substrates assembled using nucleases have excellent stablility of expression among clones. And we saw the possibility to manage individual cistron by adjusting the stength of its upstream RBS.</p>
  </div>
  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content1">
  <h2 id="application-of-hplc-in-product-detection">Apply HPLC to detect &beta;-carotene and retinol</h2>
<h3 id="cycle-1-1">Cycle 1</h3>
<h4 id="design-2">Design</h4>
<p>Prior to 2022 Team Fudan, teams aiming to synthesize &beta;-carotene conducted visual test to determine carotenoid production by looking at shades of orange. After we observed the color change of the system, for quantitative detection and validation with chemical standards, we applied HPLC to detect β-carotene and retinoids.</p>
<h4 id="build-2">Build</h4>
<p>After searching for related information, we initially decided to adopt the following protocol:<br/>
Chromatographic column: C18; mobile phase: carotene methanol:acetonitrile 70:30; retinoids methanol:acetonitrile 95:5; flow rate: 1.5 mL/min; column temperature: 40 degrees celsius; wavelength: retinal 370 nm, retinol 340 nm, carotene 454 nm (Yoon, Sang-Hwal et al. 2009).</p>
<h4 id="test-2">Test</h4>
<p>In the first few rounds of experiments, the absorption peaks of carotenoids were obvious but the absorption peaks of retinol and retinal could not be obtained.</p>
<h4 id="learn-2">Learn</h4>
<p>We should continue to consult with professionals.</p>
<h3 id="cycle-2-1">Cycle 2</h3>
<h4 id="design-3">Design</h4>
<p>We modified the protocol according to the instrument.</p>
<h4 id="build-3">Build</h4>
<p>New protocol: Agilent liquid chromatograph (HPLC-DAD); column C18 (250mm); column temperature 30°C; mobile phase methanol:water = 96:4; flow rate 0.8 mL/min; detection wavelength 325nm.<br>
The protocol is from <a target="_blank" href="https://mp.weixin.qq.com/s/81zY_LoMratSEOOy7JUaFw">Boulder Research (Guangzhou) Biotechnology Co</a>.</p>
<h3 id="test-3">Test</h3>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/beta.png" alt="beta.png (1071×336) (igem.wiki)"></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 9.</strong> 1:10 dilution of methanolic saturated solution of &beta;-carotene.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/3-7.png" alt="3-7.png (1075×353) (igem.wiki)"></p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/cww/3td7.png" alt="3td7.png (1075×362) (igem.wiki)"></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 10.</strong> Acetone extracted samples were subjected to HPLC (C18 column)as described previously. The extract from bacteria expressing CAG-MS2, BCMO, ybbO, crtYEBI (the panel above, short name 3-7) compared with the extract from bacteria expressing CAG-MS2, BCMO, ybbO-tdMCP, crtYEBI (the panel below, short name 3td7), there are two peaks in the figure between 2-4 minutes (where our standards appear <a target="_blank" href="https://mp.weixin.qq.com/s/81zY_LoMratSEOOy7JUaFw">as well others</a>, detected at 325 nm), the absorbance value of the two peaks of 3td7 in this period is significantly higher than that of 3-7.</p>
<h3 id="learn-3">Learn</h3>
<p>Clear results can be obtained for the amount of &beta;-carotene converted. Without a standard, the appropriate chromatographic separation and identification of retinol remains to challenging.</p>
  </div>
<div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content2">
<h2 id="references">References</h2>
<p>Kim, K. J., Kim, H. E., Lee, K. H., Han, W., Yi, M. J., Jeong, J., &amp; Oh, B. H. (2004). Two-promoter vector is highly efficient for overproduction of protein complexes. Protein science : a publication of the Protein Society, 13(6), 1698–1703.</p>
<p>Yoon, S. H., Lee, S. H., Das, A., Ryu, H. K., Jang, H. J., Kim, J. Y., Oh, D. K., Keasling, J. D., &amp; Kim, S. W. (2009). Combinatorial expression of bacterial whole mevalonate pathway for the production of &beta;-carotene in E. coli. <em>Journal of biotechnology</em>, <em>140</em>(3-4), 218–226.</p>
  </div>
</div>

<!-- DO NOT TOUCH THE END BELOW-->
{% endblock %}
