{% extends "layout.html" %}

{% block author_name %}Zhongyi YANG{% endblock %}
  
{% block title %}Experiments{% endblock %}
{% block lead %}Describe the research, experiments, and protocols you used in your iGEM project.{% endblock %}

{% block side_bar %}
<div id="landingpage" style="width:20%">
          <ul>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content" href="#cell-cryopreservation">
                <span class="linkname">Cell Cryopreservation</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content1" href="#cell-recovery">
                <span class="linkname">Cell Recovery</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content2" href="#colony-pcr">
                <span class="linkname">Colony PCR</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content3" href="#overlap-extension-pcr-oe-pcr">
                <span class="linkname">OE PCR</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content4" href="#agarose-gel-electrophoresis">
                <span class="linkname">Agarose Gel Electrophoresis</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content5" href="#plasmid-transformation">
                <span class="linkname">Plasmid Miniprep</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content6" href="#plasmid-miniprep">
                <span class="linkname">Plasmid Miniprep</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content7" href="#preparation-of-bacterial-cell-lysate-for-sds-page">
                <span class="linkname">Preparation of bacterial cell lysate for SDS-PAGE</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content8" href="#sds-page">
                <span class="linkname">SDS-PAGE</span></a>
              </div>
          </li>
          <li><div class="anchor">
                <a class="dot-landingpage" data-scroll="content9" href="#iptg-induction-experiment">
                <span class="linkname">IPTG induction experiment</span></a>
              </div>
          </li>
          </ul>
</div>

{% endblock %}
{% block page_content %}

<!-- ADD highlight pictures BELOW -->
<div class="row">
<br/><br/>
</div>

<!-- ADJUST PAGE CONTENT BELOW -->

<div class="row">
<p>You can view the protocols we used this year on this page.</p>

  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content">
  <h2 id="cell-cryopreservation">Cell Cryopreservation</h2>
<p>Cryopreservation is a technique that stores cells at a very low temperature (-80℃) to reduce cell metabolic damage and enable long-term storage.</p>
<ol>
<li>Add cryoprotectant: Add 80% glycerin 500 μL and bacterial fluid 1 000 μL into Cryopreservation vials. Mix upside down gently.</li>
<li>Seal the vials: Stick sellotape around the label.</li>
<li>Store the vials: Store prepared vials in the refrigerator at -80℃.</li>
</ol>
  </div>
<div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content1">
<h2 id="cell-recovery">Cell Recovery</h2>
<p>Bacteria preserved at -80℃ need to be recovered to restore cell growth.</p>
<ol>
<li>Prepare plate: Take out the plate with relevant resistance from 4 degrees and wait for the temperature to the room temperature. Mark at the bottom of the plate.</li>
<li>Recover the cell: The -80℃ frozen strain was removed and placed on the ice quickly. Take a ring of the upper layer of melted bacteria liquid with the sterilized inoculation ring. Coated the plate and draw a line.</li>
<li>Culture the recovered cell: Incubate at 37℃ overnight.</li>
</ol>
  </div>
  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content2">
  <h2 id="colony-pcr">Colony PCR</h2>
<p>PCR is performed to amplify DNA fragments in our project.</p>
<ol>
<li>
<p>Select colonies on the agar plate to pick. Culture portion of the colony in LB media with antibiotic and grow at 37 degrees. Put the remainder of the colony into a correspondingly numbered PCR tube with 20 µL of reaction mix (as below), and pipetting to complete resuspend the bacteria.</p>

<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume/μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>10x Taq Buffer</td>
<td>2</td>
</tr>
<tr>
<td>10 μM Forward Primer</td>
<td>0.5</td>
</tr>
<tr>
<td>10 μM Reverse Primer</td>
<td>0.5</td>
</tr>
<tr>
<td>Template DNA (bacteria colony)</td>
<td>added later</td>
</tr>
<tr>
<td>dNTP 10mM mix</td>
<td>0.5</td>
</tr>
<tr>
<td>Taq DNA polymerase</td>
<td>0.5</td>
</tr>
<tr>
<td>distilled water</td>
<td>16</td>
</tr>
<tr>
<td>Total</td>
<td>20</td>
</tr>
</tbody>
</table></li>
<li>
<p>Prepare PCR reaction mix before step 1, and place the mix on ice.</p>
<p>Primers used for screening crt genes insertion are:<br>
&gt; 5-crtB</p>
<p>5-ATGAATAATCCGTCGTTACTCAATCATGC-3</p>
<p>&gt; 5-crtE</p>
<p>5-ATGACGGTCTGCGCAAAAAAAC-3</p>
<p>&gt; 5-crtI</p>
<p>5-ATGAAACCAACTACGGTAATTGGTGC-3</p>
<p>&gt; 5-crtY</p>
<p>5-ATGCAACCGCATTATGATCTGATTC-3</p>
<p>&gt; rev320crtB</p>
<p>5-CCTTCCAGATGATCAAACGCGTAAG-3</p>
<p>&gt; rev320crtE</p>
<p>5-ATGAGAATGAATGGTAGGGCGTC-3</p>
<p>&gt; rev320crtI</p>
<p>5-GGATTAAACTGCTGAATCTGCGCTTC-3</p>
<p>&gt; rev320crtY</p>
<p>5-CCGCGGTATCCATCCACAAG-3</p>

<table>
<thead>
<tr>
<th>Temperature</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>94℃</td>
<td>05:00</td>
</tr>
<tr>
<td>94℃</td>
<td>00:30</td>
</tr>
<tr>
<td>60℃</td>
<td>00:30</td>
</tr>
<tr>
<td>72℃</td>
<td>03:30, cycle 3-4-5 for 25 times</td>
</tr>
<tr>
<td>72℃</td>
<td>05:00</td>
</tr>
<tr>
<td>16℃</td>
<td>∞</td>
</tr>
</tbody>
</table></li>
<li>
<p>The PCR program above is for screening crt genes insertion during the 2022 iGEM season.</p>
</li>
<li>
<p>Agarose gel electrophoresis of PCR products. One lane on the gel is loaded with 6 uL D2000 DNA ladder. Run the gel at 100 V for 20-30 minutes, and examine the gel under UV.</p>
</li>
<li>
<p>The correct bacterial clones are sent for Sanger sequencing. Once verified, these clones would be used for further experiments.</p>
</li>
</ol>
  </div>
  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content3">
  <h2 id="overlap-extension-pcr-oe-pcr">Overlap Extension PCR (OE PCR)</h2>
<p>OE PCR is used to fuse fragments.</p>
<ol>
<li>
<p>First-round PCR: Use primer a/b to amplify fragment AB by PCR, and c/d to amplify fragment CD. Set up PCR system:</p>

<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume/μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>distilled water</td>
<td>To 25</td>
</tr>
<tr>
<td>2 x Phanta Max Buffer</td>
<td>12.5</td>
</tr>
<tr>
<td>dNTP</td>
<td>0.5</td>
</tr>
<tr>
<td>Phanta DNA polymerase</td>
<td>0.5</td>
</tr>
<tr>
<td>DNA Template</td>
<td>Plasmid: 0.5<br>PCR product: 1</td>
</tr>
<tr>
<td>Forward Primer</td>
<td>1</td>
</tr>
<tr>
<td>Reverse Primer</td>
<td>1</td>
</tr>
</tbody>
</table></li>
<li>
<p>Set up reaction program:</p>

<table>
<thead>
<tr>
<th>Procedure</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initialization</td>
<td>95℃</td>
<td>30s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95℃</td>
<td>15s</td>
<td>30-40</td>
</tr>
<tr>
<td>Annealing</td>
<td>depends on the primer</td>
<td>15s</td>
<td></td>
</tr>
<tr>
<td>Extension</td>
<td>72℃</td>
<td>30s/kb</td>
<td></td>
</tr>
<tr>
<td>Final elongation</td>
<td>72℃</td>
<td>5min</td>
<td>1</td>
</tr>
</tbody>
</table></li>
<li>
<p>Annealing of homologous regions in Second round PCR: Separate and purify fragment AB and CD by agarose gel electrophoresis and DNA gel extraction. Measure the concentration of the two fragments. Dilute them into 1:1 as a template. Set up PCR system:</p>

<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>distilled water</td>
<td>To 50μL</td>
</tr>
<tr>
<td>2 x Phanta Max Buffer</td>
<td>25μL</td>
</tr>
<tr>
<td>dNTP</td>
<td>1μL</td>
</tr>
<tr>
<td>Phanta</td>
<td>1μL</td>
</tr>
<tr>
<td>Template</td>
<td>0.08 pmol + 0.08 pmol</td>
</tr>
</tbody>
</table></li>
<li>
<p>Set up reaction program:</p>

<table>
<thead>
<tr>
<th>Procedure</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initialization</td>
<td>94℃</td>
<td>5min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94℃</td>
<td>30s</td>
<td>12</td>
</tr>
<tr>
<td>Annealing</td>
<td>60/64/68/72℃</td>
<td>30s</td>
<td></td>
</tr>
<tr>
<td>Extension</td>
<td>72℃</td>
<td>1min/kb</td>
<td></td>
</tr>
<tr>
<td>Final elongation</td>
<td>72℃</td>
<td>7min</td>
<td>1</td>
</tr>
</tbody>
</table></li>
<li>
<p>Store the vials: Store prepared vials in the refrigerator at -80℃.</p>
</li>
</ol>
  </div>
    <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content4">
    <h2 id="agarose-gel-electrophoresis">Agarose Gel Electrophoresis</h2>
<p>Agarose gel electrophoresis is performed to separate and confirm whether our plasmids were constructed properly.</p>
<ol>
<li>Make gel solution: Add 0.7g agarose, 70ml TAE buffer into a glass bottle and heat in a micro-oven for 2 min. Cool the liquid agarose gel to lower than 60℃ and decant the liquid agarose gel into an agarose gel tank. Add 8 μL EB into the liquid agarose gel and place the electrophoresis comb.</li>
<li>Load the gel: Place the solid agarose gel in an electrophoresis device. Add 10xDNA Loading Buffer in the DNA sample, mix them up gently, and carefully pipette the sample into the sample loading chambers.</li>
<li>Electrophoresis: Cover the lid of the electrophoresis device, set the electrophoresis time, and start electrophoresis.</li>
</ol>
    </div>
    <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content5">
    <h2 id="plasmid-transformation">Plasmid transformation</h2>
<p>Plasmid transformation is performed to transfer plasmids into the host bacteria. We transform our plasmids into DH5α <em>E. coli</em> to amplify them and BL21 to testify function of our plasmids.</p>
<ol>
<li>Thaw all reagents on ice. And add 20 μL competent <em>E. coli</em> cells into the ligation product.</li>
<li>Heat shock at 42℃ for 45s and then cool the mixture on ice for 2min.</li>
<li>Add 900 μL liquid SOC or LB medium (without antibiotic) into the mixture and shake culture at 37℃ for 1h.</li>
<li>Evenly spread the liquid culture on a solid culture medium and incubate at 37℃ overnight for colonies forming on the plate.</li>
</ol>
    </div>
<div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content6">
<h2 id="plasmid-miniprep">Plasmid Miniprep</h2>
<p>To extract and purify plasmids, we carried out this based on the Vazyme Mini-prep Kit manual.</p>
<ol>
<li>Pellet bacteria culture in a 1.6mL centrifugation tube at 13,000rpm for 1 minute. Decant supernatant. The pellet in the tube can be stored in -20 freezer if time not enough for the following steps.</li>
<li>Add 250uL of resuspension solution (P1 with RNase) and vortex. After adding RNase into the P1, P1 is stored in 4 degree refrigerator.</li>
<li>Add 250uL of lysis solution (P2) and invert gently 4-6 times. P2 contains NaOH, and must keep tightly closed.</li>
<li>Add 350uL of neutralization solution (P3) and invert gently 4-6 times. Do not vortex, which will break bacteria genomic DNA and genomic DNA fragments will contaminate purified plasmids.</li>
<li>Spin at 13,000rpm for 10 minutes, at 4 dgree.</li>
<li>Take the clear supernatant, without any white protein cloud. Bind the DNA to the column by decanting.</li>
<li>Centrifuge at 13,300 rpm for 1 minute.</li>
<li>Wash the column with 600uL of wash solution (PW2) and centrifuge for 1 minute. Remind adding pure ethanol into PW2 as indicated on the bottle.</li>
<li>Discard the flow-through, wash again with 600uL of PW2, and centrifuge for 1 minute. Discard the flow-through and centrifuge the column again for 2 minutes.</li>
<li>Keep the cap open and wait until no ethanol smell. Move the column into a new centrifugation tube. Elute the purified DNA with 50uL of elution buffer (TE; 10 mM Tris-Cl, 1 mM EDTA). Then, centrifuge for 1 minutes. Collect the flow-through, which is the plasmid DNA. Because pET28 based is low-copy plasmid, pre-warm TE to 50 degree helps the elution.</li>
</ol>
</div>
<div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content7">
<h2 id="preparation-of-bacterial-cell-lysate-for-sds-page">Preparation of bacterial cell lysate for SDS-PAGE</h2>
<ol>
<li>Grow the cells overnight, to an OD600 above 1. If IPTG induction needed, at it around OD600 0.2-0.3 (keep IPTG stock solution at -20 freezer).</li>
<li>Take one 1 ml of OD600 = 1 from each sample. Correct the sample volume to obtain an equivalent size pellet. For us, as quantified using NanoCym950 nanoparticles (equivalent to the size of <em>E. coli</em>), 1 OD600 equals to 10^8 nanoparticles per mL. We estimate each well on SDS-PAGE contains proteins from 2*10^6 bacteria.</li>
<li>Spin down the bacterial cells for 1 min at 13,000 rpm in a microfuge.</li>
<li>Aspirate any trace of supernatant with a vacuum line avoiding touching the pellet, but removing all liquid.</li>
<li>To each cell pellet, add 250 µL of distilled water, resuspend the pellet. Add 250 µL 2x SDS sample buffer (see below). Heat with a 95 degree heating block for 10 minutes. Store the samples at -20 degree freezer until SDS-PAGE.</li>
<li>2x SDS sample buffer, works for Tris-Cl or MOPS based gels
<ul>
<li>Glycerol 20 ml</li>
<li>SDS 4 g</li>
<li>0.5 M Tris-HCl pH 6.8 stock solution 25 ml</li>
<li>Bromo Phenol Blue 1 mg; dissolve and volume to 95 ml</li>
<li>add 5% β-Mercaptoethanol before usage</li>
</ul>
</li>
</ol>
</div>
<div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content8">
<h2 id="sds-page">SDS-PAGE</h2>
<p>SDS-PAGE is performed for the separation of polypeptides and confirms whether our circuits expressed properly.</p>
<ol>
<li>Prepare 10ml 10% Running Gel solution: Add4.1 mL ddH2O, 3.3 mL 30% Acrylamide/Bis (29:1 or 37.5:1), 2.5 mL 1.5 M Tris-HCl pH8.8, 100 μL 10% SDS, 50 μL 10% APS, 5 μL TEMED. Mix them thoroughly.</li>
<li>Prepare 2ml 4% Stacking Gel solution: Add 1.22 ml ddH2O, 0.26ml 30% Acrylamide/Bis (29:1 or 37.5:1), 0.5 mL 0.5 M Tris-HCl pH6.8, 20 μL 10% SDS, 20 μL 10% APS, 2 μL TEMED. Mix them thoroughly.</li>
<li>Prepare the protein sample: Add 1 ml bacterial cells per time into a centrifuge tube, centrifuge at 12,000 x g for 1min, discard the supernatant. Dilute SDS sample buffer to 1x. Add 1 x SDS Loading to 3000 cells per 1 μL, mix thoroughly by vortex or pipetting up and down until there is no visible precipitation. Incubate at 99℃ for 5min.</li>
<li>Make the gel: Assemble the gel cassette and make sure it does not leak. Fill the gel cassette with the Running Gel softly and fill up the cassette with ddH2O. Keep it still for 10 - 20min until the water layer can be observed. Pour out the ddH2O completely and fill up the cassette with Stacking Gel. Insert the comb and take care not to catch bubbles under the teeth. Keep it still.</li>
<li>Load the gel: Take off the cassette and assemble the gel running stand. Fill the stand with 1 x SDS running buffer and remove the combs from the gel. Mix up Marker with 1 x SDS Loading. Load 15 μL marker mixture into the wells.</li>
<li>Electrophoresis: Cover the lid of the electrophoresis device, and start electrophoresis at 200 V until the dye front is nearly at the bottom of the gel.</li>
<li>Stain the gel: Submerge the whole piece of the disassembled gel with 0.1% Coomassie Blue dye for 30min.</li>
<li>Destain the gel: Pour out the 0.1% Coomassie Blue dye and wash it using ddH2O. Destain with a destaining solution for 30min. Change the destaining solution to destain until it is clear.</li>
<li>Scan the gel.</li>
</ol>
</div>
<div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content9">
<h2 id="iptg-induction-experiment">IPTG induction experiment</h2>
<ol>
<li>Reconnection: Take 10 mL and 5 mL LB liquid medium in two test tubes, 10 mL one is used as the control group and 5 mL one is for the experiment group. Add 500× ampicillin 20 μL and 10 μL, respectively. Culture 6h. Add 1 mL bacteria solution to the experiment group and divide evenly into two tubes (3 mL each) for IPTG-induced and non-induced groups. Another 1 mL bacteria was used to measure initial OD.</li>
<li>Measure OD: Measured OD value once per 30min and once per 15min after OD reached 0.2.</li>
<li>Induce: When OD value reached about 0.6, add 500mM IPTG to reach the final concentration of 1 mM. Incubate for 3 hours.</li>
</ol>
<h2 id="preparation-of-agarose-pad-for-microscopy">Preparation of agarose pad for microscopy</h2>
<p>Click <a href="https://www.protocols.io/view/prepare-agarose-pad-for-microscopy-2022-igem-8epv5jze5l1b/v1">here</a> to view our steps to perform fluorescence microscopy for <em>E. Coli</em></p>
</div>
<div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content9">
<h2 id="acetone-extraction-of-bacteria-pellet-for-hplc">Acetone extraction of bacteria pellet for HPLC</h2>
<p>Click <a href="https://www.protocols.io/view/acetone-extraction-of-bacteria-pellet-for-hplc-202-eq2ly7peelx9/v1">here</a> to view how we applied HPLC on bacterial products.</p>
</div>
</div>

<!-- DO NOT TOUCH THE END BELOW-->
{% endblock %}
