{% extends "layout.html" %}

{% block author_name %}the Team{% endblock %}
  
{% block title %}Measurement{% endblock %}
{% block lead %}Synthetic Biology needs great measurement approaches for characterizing parts, and efficient new methods for characterizing many parts at once. Describe your measurement approaches on this page.{% endblock %}

{% block side_bar %}
<div id="landingpage" style="width:20%">
          <ul>
          <li><div class="anchor">
          <a class="dot-landingpage" data-scroll="content" href="#overview">
          <span class="linkname">Overview</span></a>
          </div>
        <ul class="submenu blank">
            <li>
                <a class="anchor linkname" href="#overview">Overview</a>
            </li>
        </ul>
        </li>
        <li>
            <div class="layer">
              <div class="anchor">
                <a class="dot-landingpage" data-scroll="content1" href="#verification">
                <span class="linkname">Verification</span></a>
                </div>
              <img src="https://static.igem.wiki/teams/4162/wiki/arrow-orange.png" class="arrow" style="cursor:pointer"></img>
            </div>
        <ul class="submenu">
            <li>
                <a class="anchor linkname" href="#verification">Verification</a>
            </li>
            <li>
                <a class="anchor" href="#pcr-to-confirm-the-successful-transformation-of-plasmids-into-e.coli">PCR</a>
            </li>
            <li>
                <a class="anchor" href="#sds-page-to-confirm-the-successful-expression-of-proteins-and-the-effects-of-rbs-intensity-on-protein-expression">SDS-PAGE</a>
            </li>
            <li>
                <a class="anchor" href="#color-changes-to-verify-β-carotene-production">Visual Detection</a>
            </li>
            <li>
                <a class="anchor" href="#fluorescence-microscopy-to-confirm-the-successful-expression-of-membrane-proteins">Fluorescence microscopy</a>
            </li>
            <li>
                <a class="anchor" href="#hplc-for-detection-of-retinoids">HPLC</a>
            </li>
        </ul>
          </li>
        <li>
            <div class="layer">
              <div class="anchor">
                <a class="dot-landingpage" data-scroll="content2" href="#optimization">
                <span class="linkname">Optimization</span></a>
                </div>
              <img src="https://static.igem.wiki/teams/4162/wiki/arrow-orange.png" class="arrow" style="cursor:pointer"></img>
            </div>
        <ul class="submenu">
            <li>
                <a class="anchor linkname" href="#optimization">Optimization</a>
            </li>
            <li>
                <a class="anchor" href="#verify-that-adjusting-rbs-intensity-promotes-metabolism">Adjusting RBS</a>
            </li>
            <li>
                <a class="anchor" href="#fluorescence-microscopy-to-confirm-the-successful-expression-of-membrane-proteins">Membrane protein</a>
            </li>
        </ul>
          </li>
        <li><div class="anchor">
          <a class="dot-landingpage" data-scroll="content3" href="#conclusion">
          <span class="linkname">Summary</span></a>
          </div>
        <ul class="submenu blank">
            <li>
                <a class="anchor linkname" href="#conclusion">Summary</a>
            </li>
        </ul>
        </li>
        <li><div class="anchor">
          <a class="dot-landingpage" data-scroll="content4" href="#references">
          <span class="linkname">References</span></a>
          </div>
        <ul class="submenu blank">
            <li>
                <a class="anchor linkname" href="#references">References</a>
            </li>
        </ul>
        </li> 
        </ul>
</div>
<!-- hide sidebar -->
{% endblock %}
{% block page_content %}

<!-- ADD highlight pictures BELOW -->
<div class="row">
<br/><br/>
</div>

<!-- ADJUST PAGE CONTENT BELOW -->

<div class="row">
  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content">
  <h2 id="overview">Overview</h2>
  <p>Within a single-cell bacterial factory, exogenous genes are expressed to modify metabolites. <strong>Co-expression</strong> is often achieved by <strong>co-transforming</strong> two or more plasmids, each carrying the gene of one subunit and a different selection marker, into the heterologous expression host. However, the number of plasmids used is not infinite, and simultaneously maintaining more than four plasmids is very challenging.</p>
<p>The <strong>uneven copy numbers</strong> of each plasmid make protein expression and metabolite production unpredictable. Another approach is to <strong>assemble multiple genes into one plasmid</strong> where genes are transcribed from multiple promoters (1) or a single promoter to produce long lines of cis mRNA (2).</p>
<p>Polycistronic expression suffers from a major problem: translations of cistrons are severely unbalanced. The order of cistrons in the polycistronic mRNA has an unpredictable consequence on the levels of protein expression (3). <strong>Progressively lower expression was reported with more downstream cistron in the mRNA</strong> (4).</p>
<p>The must step to validate our/your engineering design works is not until <strong>the target product is present and detected</strong>. We integrated a whole set of measurement approaches to validate <a href="/fudan/parts">our parts</a>, which can be applied to other occasions as well.</p>
  </div>

  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content1">
    <h2 id="verification">Verification</h2>
<h3 id="pcr-to-confirm-the-successful-transformation-of-plasmids-into-e.coli">PCR to confirm the successful transformation of plasmids into <i>E. coli</i></h3>
<p>Before we verify protein expression we need to confirm whether <strong>the target gene has been successfully transformed into the plasmids</strong>.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/syj/iyeb-dna.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 1.</strong> Agarose gel electrophoresis of PCR products, amplified from a bacterial colony.</p>
<p>The first lane was loaded with a D2000 DNA ladder whose sizes were marked on the image. We chose Taq DNA polymerase for its low cost and high reliability, and we designed forward and reverse primers for each carotene synthesis enzyme (crt for short). The PCR reaction was composed of 2 μL 10x Taq polymerase buffer, 16 μL H<sub>2</sub>O, 0.5 μL Taq polymerase, 0.5 μL dNTP (10 mM each), 0.5 μL forward primer (10 mM), 0.5 μL reverse primer (10 mM), and 1 colony picked from the plate. Using the same forward primer, and different reverse primers, we were able to detect the composition of various crt genes. After PCR, the correct bacterial clones were sent for Sanger sequencing, which take 24-48 hours. While waiting, we start preparing the following experiments. Once the sequencing confirmed, we execute the next.</p>

<h3 id="sds-page-to-confirm-the-successful-expression-of-proteins-and-the-effects-of-rbs-intensity-on-protein-expression">SDS-PAGE to confirm the successful expression of proteins and the effects of RBS intensity on protein expression</h3>
<p><img src="https://static.igem.wiki/teams/4162/wiki/syj/3a1.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 2.</strong> SDS-PAGE of whole cell lysate of bacterial cultures.</p>
<p>Protein expression was induced in parallel cultures by IPTG. Bacterial cultures were monitored by OD600, and 5x10^7 cells were harvested by centrifugation and lysis in 1x SDS sample buffer. An equal amount (10 μL, 2x10^6 cells) of whole cell lysate was analyzed by SDS-PAGE (4~20% gradient gel, Tanon brand). Red arrows point to crtI protein. Green arrows point to crtY protein. Black arrows point to crtB protein. Yellow arrows point to crtE protein.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/syj/b.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 3.</strong> We failed to see IPTG induced expression of the membrane protein BCMO on the protein gel, so we turned to microscopy by tagging fluorescent protein downstream the target protein, which is included <a href="#fluorescence-microscopy-to-confirm-the-successful-expression-of-membrane-proteins">on this page</a>.</p>

<h3 id="color-changes-to-verify-β-carotene-production">Color changes to verify β-carotene production</h3>
<p>As our intermediate product is the yellow β-carotene, we can confirm its presence simply by observing the color change in the bacterial pellet, sometime even the liquid bacterial culture. It served as the visual assay to tell whether the four crt enzymes cooperated effectively under the same or different types of RBS.</p>
<p>No color change in the bacterial pellet along with the unequal protein expression induced by IPTG pushed us to adjust the strength of RBS in front of each enzyme. By enhancing RBS or weakening it, we successfully obtained the yellow products, as shown in <a href="http://parts.igem.org/Part:BBa_K4162117" target="_blank">BBa_K4162117</a> and <a href="http://parts.igem.org/Part:BBa_K4162118" target="_blank">BBa_K4162118</a>.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/syj/&beta;-carotene.jpeg" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 4.</strong> Bacterial pellets, after acetone extraction (in-complete), still bear the product β-carotene yellow.</p>

<h3 id="fluorescence-microscopy-to-confirm-the-successful-expression-of-membrane-proteins">Fluorescence microscopy to confirm the successful expression of membrane proteins</h3>
<p>Failure in observing the expression of membrane protein BCMO from SDS-PAGE pushed us to reflect and try to find out which step went wrong. We could not any band on the protein gel after induction by IPTG in BL21(DE3) strain, and we confirmed that we are using the same design as reported (5). This situation set us thinking maybe it should be the assay’s fault. After consulting Prof. Motoyuki Hattori, who has extensive knowledge in expressing, purifying membrane protein, we knew that SDS-PAGE with CBB staining is not suitable for membrane protein expression detection, which is always below the detection. So, we changed to this fluorescence microscopy assay.</p>

<p><img src="https://static.igem.wiki/teams/4162/wiki/figure-2-cell-example.jpg" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 5.</strong> Fluorescent imaging confirm the expression of both BCMO and ybbO.
We observed fluorescence images of <i>E. coli</i> (1. Bacteria expressing <a target="_blank" href="http://parts.igem.org/Part:BBa_K4162031" target="_blank">BCMO-linker-StayGold</a>; 2. Bacteria expressing <a href="http://parts.igem.org/Part:BBa_K4162029" target="_blank">BCMO and ybbO-tdMCP-GFP</a>) through an Olympus fluorescence microscope with a 1.45 NA 150× oil objective. GFP and StayGold were excited by a 488 nm laser. Images were taken by the same settings for comparison. For details about sample preparation, please check our protocol <a target="_blank" href="https://www.protocols.io/view/prepare-agarose-pad-for-microscopy-2022-igem-8epv5jze5l1b/v1">here</a>.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/figure-1-fluorescence-intensity.jpg" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 6.</strong> Fluorescent imaging revealed that IPTG induction increases the expression of both BCMO and ybbO.</p>
<p style="font-size:0.8rem;color:grey">We observed fluorescence images of Hi-control Rosetta strain (1. Bacteria expressing BCMO-linker-StayGold; 2. Bacteria expressing BCMO and ybbO-tdMCP-GFP) through an Olympus fluorescence microscope. Samples induced with IPTG have stronger signal than controls.</p>
<h3 id="hplc-for-detection-of-retinoids">HPLC for detection of retinoids</h3>
<p>After we observed the color change of the system, for quantitative detection and validation with chemical standards, we applied HPLC to detect β-carotene and retinoids.</p>
<p><img src="https://static.igem.wiki/teams/4162/wiki/syj/hplc.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 7.</strong> Acetone extracted samples were subjected to HPLC (C18 column) as described previously (5), except we are using <a href="https://mp.weixin.qq.com/s/81zY_LoMratSEOOy7JUaFw" target="_blank">methanol:water = 96:4</a> as the moving phase. The extract from bacteria DH5&alpha; expressing CAG-MS2, BCMO, ybbO, crtYEBI (the panel above, short name 3-7) compared with the extract from bacteria expressing CAG-MS2, BCMO, ybbO-tdMCP, crtYEBI (the panel below, short name 3td7), there are two peaks in the figure between 2-4 minutes (where our standards appear <a href="https://mp.weixin.qq.com/s/81zY_LoMratSEOOy7JUaFw" target="_blank">as well others</a>, detected at 325 nm), the absorbance value of the two peaks of 3td7 in this period is significantly higher than that of 3-7, which depicted <strong>the efficiency of TEARS</strong> in improving the production of retinol.</p>

  </div>

  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content2">
    <h2 id="optimization">Optimization</h2>
<h3 id="verify-that-adjusting-rbs-intensity-promotes-metabolism">Verify that adjusting RBS intensity promotes metabolism</h3>
<p>In our project, we used ribozyme to separate our crtEBIY biobrick when it produced <i>in vivo</i>. With the help of self-cleaving ribozyme, self-interaction of the polycistron can be avoid and the translation of each cistron should be simply dependent on the strength of its upstream RBS.</p>

<p><img src="https://static.igem.wiki/teams/4162/wiki/syj/d.png" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 8.</strong> Proof the efficiency of ribozyme-assisted polycistronic expression.</p>

<p>When the RBS strength of the four enzymes in the our linear reaction was consistent, the results on the protein gel showed that the expression of upstream proteins (crtI and crtY) was higher than that of downstream proteins (crtE and crtB) after IPTG induction, which was not in line with our expectation of the equal consistent level of the four enzymes. Althrough we use the leaky expression of the promoter, IPTG result suggests that not only RBS but also the protein sequence affects the yield. Moreover, the bacteria pellet was not yellowing, indicating that we do not have the ideal product β-carotene.</p>

<p><strong>When we weakened the RBS strength of one upstream enzyme (crtY)</strong>, the protein gel (Figure 8) showed the disappear of strong crtY band, and the bacterial culture as well as the pellet turned yellow, indicating that we had obtained β-carotene. Similar result was obtained when we <a href="http://parts.igem.org/Part:BBa_K4162118" target="_blank">enhanced</a> the expression of one downstream enzyme (crtE).</p>

<p><img src="https://parts.igem.org/wiki/images/thumb/8/83/T--Fudan--Hi--117.png/800px-T--Fudan--Hi--117.png.jpeg" alt=""></p>
<p style="font-size:0.8rem;color:grey"><strong>Figure 9.</strong> IPTG(-/+) = without/with 0.2 mM IPTG for 3-6 hours, adding IPTG to a bacteria culture with OD600 0.2-0.3. M: Protein molecular weight marker ladder. Lane 1~2: pET28 plasmids encoding crtEBIY without any tag were transformed into BL21(DE3) Rosetta strain, single clones (6b) were picked for liquid LB culture. Lane 3~12: pET28 plasmids encoding crtYEBI without any tag were transformed into BL21(DE3) Hi-Control strain, single clones (Hi-7, Hi-7a, Hi-7b, Hi-7c, Hi-7d) were picked for liquid LB culture. Lane 14~17: pET28 plasmids encoding crtB, crtE, crtI, crtY without any tag were transformed into BL21(DE3) Hi-Control strain, single clones (1B1, E1, I7, Y1) were picked for liquid LB culture. Protein expression was induced in parallel cultures by IPTG. Bacterial cultures were monitored by OD600, and 5x10^7 cells were harvested by centrifugation and lysis in 1x SDS sample buffer. Equal amount (10 μL, 2x10^6 cells) of whole cell lysate were analyzed by SDS-PAGE (4~20% gradient gel, Tanon brand). Red arrows point to crtI protein. Green arrows point to crtY protein. Black arrows point to crtB protein. Yellow arrows point to crtE protein.</p>

<p>By <strong>combining protein gel</strong> with <strong>the observation of culture/pellet color changes</strong>, we could verify that <em><strong>adjusting the RBS strength of individual mono-cistrons in the presence of ribozyme in polycistronic plasmids could direct linear reactions to favrite metabolite production</strong></em>.</p>

<!-- NOT USE, ALREADY METIONED    h3 id="membrane-protein-detection">Membrane protein detection</h3>
<p>Initially we chose to detect the expression level of the membrane protein BCMO in DH5&alpha;  and Hi-control BL21(DE3) by SDS-PAGE with CBB staining. However, we came to realize that it is not suitable for membrane protein expression detection, attributed to the degradation of membrane proteins as they are separated from the membrane.</p>
<p>To detect the expression of membrane proteins without affecting their activity, we tagged a segment of fluorescent protein behind the membrane proteins sequence. In this way, <strong>it is not necessary to lyse the bacteria and destroy the protein structure, and the conclusion that BCMO is successfully expressed can be obtained by observing the fluorescent substances under the microscope.</strong></p -->
  </div>

  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content3">
    <h2 id="conclusion">Summary</h2>
<p>We integrated a series of common but robust measurements to <strong>step-by-step verify and optimze our engineered bacteria</strong>. We verify the linear reaction step by step, and employ <strong>error correction and optimization</strong>, so as to drive bacterial metabolism to the direction we designed, to ensure the efficiency of the system and get the final product. On the basis of the detection results, the reaction was optimized by <strong>adjusting RBS strength</strong> and introducing <strong>TEARS</strong>, a recent published (6) bacterial enzyme enrichment tool, to <strong>increase the yield for further commercial use</strong>.</p>
  </div>
  <div class="offset-md-3 col-md-7 col-sm-12 my-lg-5 content section" id="content4">
    <h2 id="references">References</h2>
<ol>
<li>Cheich, C., Kummel, D., Soumailakakis, D., Heinemann, U. and Bussow, K. (2007) Vectors for co-expression of an unrestricted number of proteins. Nucleic Acids Res, 35, e43.
</li>
<li>Tan, S., Kern, R.C. and Selleck, W. (2005) The pST44 polycistronic expression system for producing protein complexes in Escherichia coli. Protein Expr Purif, 40, 385-395.
</li>
<li>Renaud-Gabardos, E., Hantelys, F., Morfoisse, F., Chaufour, X., Garmy-Susini, B. and Prats, A.C. (2015) Internal ribosome entry site-based vectors for combined gene therapy. World J Exp Med, 5, 11-20.
</li>
<li>Kim, K.J., Kim, H.E., Lee, K.H., Han, W., Yi, M.J., Jeong, J. and Oh, B.H. (2004) Two-promoter vector is highly efficient for overproduction of protein complexes. Protein Sci, 13, 1698-1703.
</li>
<li>Jang, H. J., Yoon, S. H., Ryu, H. K., Kim, J. H., Wang, C. L., Kim, J. Y., Oh, D. K., &amp; Kim, S. W. (2011). Retinoid production using metabolically engineered Escherichia coli with a two-phase culture system. Microbial cell factories, 10, 59.
</li>
<li>Spatial engineering of E. coli with addressable phase-separated RNAs.
Guo H, Ryan JC, Song X, Mallet A, Zhang M, Pabst V, Decrulle AL, Ejsmont P, Wintermute EH, Lindner AB.
Cell, 2022 Sep 29;185(20):3823-3837. e23.
</li>
</ol>
  </div>
</div>

<!-- DO NOT TOUCH THE END BELOW-->
{% endblock %}
